tests/test_core.py

```diff
@@ -5010,6 +5010,9 @@ def test_utf32(self):
     self.do_runf(path_from_root('tests', 'utf32.cpp'), 'OK.')
     self.do_runf(path_from_root('tests', 'utf32.cpp'), 'OK.', args=['-fshort-wchar'])
 
+  def test_utf16(self):
+    self.do_runf(path_from_root('tests', 'utf16.cpp'), 'OK.')
```

Comment:
Please put this new file in `tests/core` and name it `test_utf16.cpp` to match the test name (I see that doesn't match the tests right above but its the preferred convention in this file).

Reply:
Thanks, will do.